docs/blog/lit-element-deepdive-in-batching-updates/index.md

```diff
@@ -0,0 +1,615 @@
+---
+title: 'LitElement: A deep dive into batching updates'
+pageTitle: 'LitElement: A deep dive into batching updates'
+date: 2020-12-28
```

Comment:
```suggestion
date: 2020-12-26
```

so that we don't have 2 blogs at the same date

Reply:
I've not much mind, but should this match the date on dev.to?